.github/workflows/test_release_conda.yml

```diff
@@ -0,0 +1,68 @@
+name: Test Conda-forge release
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'The Euphonic release version to test e.g. 0.6.1'
+        required: true
+        type: string
+
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The Euphonic release version to test e.g. 0.6.1'
+        required: true
```

Comment:
Should this match?
```suggestion
        required: true
        type: string
```

Reply:
Weirdly, `type` is a required parameter for workflow call but not for workflow dispatch. I guess there is no harm in having it on both.